### ISSUE TITLE
DatesDatesDates plugin creates aria-describedby broken reference issue

### ISSUE THREAD

**Issue by drn05r** [eprints/eprints3.4#157]
The DatesDatesDates plugin adds the 'dates' field that cannot define the correct help element for the aria-describedby reference.  Due to the use of inner and outer suffixed help elements for this configuration of field.
